ADALiOS/ADALiOS/ADAL.h

```diff
@@ -19,12 +19,10 @@
 
 #import "ADLogger.h"
 #import "ADAuthenticationContext.h"
-#import "ADTokenCacheStoring.h"
 #import "ADAuthenticationError.h"
 #import "ADAuthenticationResult.h"
-#import "ADTokenCacheStoreItem.h"
+#import "ADTokenCacheItem.h"
 #import "ADUserInformation.h"
-#import "ADTokenCacheStoreKey.h"
```

Comment:
Cache key is public interface supported by both .NET and Android. We should not remove it.